src/util.js

```diff
@@ -322,25 +322,15 @@ define([
             }
             return value;
         }
-        opts = opts || {};
-        if (opts.normalize_xmlns) {
-            var xmlns = $(xml.parseXML(serverForm)).find('data').xmlAttr('xmlns');
-            localForm = localForm.replace(/(data[^>]+xmlns=")(.+?)"/,
-                                    '$1' + xmlns + '"');
-        }
         localForm = cleanForDiff(localForm);
         serverForm = cleanForDiff(serverForm);
-        var patch = jsdiff.createPatch(
-            "",
-            serverForm,
-            localForm,
-            gettext("Server Form"),
-            gettext("Local Form")
-        );
-        patch = patch.replace(/^Index:/,
-            opts.not ? gettext("XML should not be equivalent") : gettext("XML mismatch")
-        );
-        return patch;
+
+        const dmp = new diff_match_patch(),
+            diff = dmp.diff_main(localForm, serverForm);
+
+        let html = dmp.diff_prettyHtml(diff);
+        html = html.replaceAll("&para;<br>", "<br>");
```

Comment:
Is this effectively you overriding the output of `prettyHtml` based on preference? If so, is it worth noting that or does that seem clear as is?

Reply:
Good point, this is a good place for a comment - 321bc1da

This is how things look without doing this replacement 😱 
<img width="353" alt="Screenshot 2025-04-23 at 4 17 46 PM" src="https://github.com/user-attachments/assets/22bf6ed3-348c-44e9-ad66-1ddeea2739b0" />